index.js

```diff
@@ -0,0 +1,2 @@
+require('./format-as-currency');
```

Comment:
i added umd support in the https://github.com/bcherny/format-as-currency/tree/umd branch. can you give that a try and let me know if it works for your use case?